lib/llm/message.rb

```diff
@@ -2,11 +2,12 @@
 
 module LLM
   class Message
-    attr_accessor :role, :content
+    attr_accessor :role, :content, :logprobs, :top_logprobs
 
-    def initialize(role, content)
+    def initialize(role, content, logprobs: nil)
```

Comment:
Why don't we just pass the completion into the message ? Then we can build features on top of it, including `logprobs` and more. Plus the response would be available to the caller / conversations. Picking and choosing bits of the response doesn't seem as useful. We're essentially hiding information from the caller this way, and only providing these select bits of information that we choose. No bueno.